interpreter/cling/include/cling/Interpreter/Transaction.h

```diff
@@ -463,8 +463,8 @@ namespace cling {
         m_NestedTransactions->clear();
     }
 
-    llvm::Module* getModule() const { return m_Module.get(); }
-    void setModule(std::unique_ptr<llvm::Module> M) { m_Module.swap(M); }
+    std::shared_ptr<llvm::Module> getModule() const { return m_Module; }
```

Comment:
`const&`, if possible.

Reply:
When the `shared_ptr` is used as a return type, RVO and automatic moves make it no different.